pkgs/build-support/writers/default.nix

```diff
@@ -240,9 +240,9 @@ rec {
       then "${python}/bin/python"
       else "${python.withPackages (ps: libraries)}/bin/python"
     ;
-    check = writeDash "python2check.sh" ''
+    check = if python.isPy3k then writeDash "python2check.sh" ''
```

Comment:
maybe we should name it pythonckeck.sh ?

Reply:
and this could use lib.optionalString to avoid the ``else ""``.

Reply:
Thanks, updated.